v2/tools/generator/internal/codegen/pipeline/crossplane_add_owner_properties.go

```diff
@@ -112,7 +112,7 @@ func lookupOwners(defs astmodel.TypeDefinitionSet, resourceDef astmodel.TypeDefi
 		}, nil
 	}
 
-	owner := resourceType.Owner()
+	owner := resourceType.Owner().(astmodel.InternalTypeName)
```

Comment:
Is this changing at some point to be `InternalTypeName` as well and thus not need the cast?

Reply:
Yes.